database/migrations/2025_07_15_171542_create_flights_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('origin_id')->constrained('cities');
+            $table->foreignId('destination_id')->constrained('cities');
+            $table->foreignId('airline')->constrained('airlines');
+            $table->dateTime('departure_time');
+            $table->dateTime('arrival_time');
```

Comment:
```suggestion
            $table->dateTime('departure_at');
            $table->dateTime('arrival_at');
```
"time" refers to hours/minutes/seconds. "at" refers to that + the date.
```
09:25:00 // time

2025-07-16 09:25:00 // at
```

Reply:
Didn't know that, changed it